repository/src/main/java/org/alfresco/service/cmr/repository/ContentService.java

```diff
@@ -170,4 +173,72 @@ public ContentWriter getWriter(NodeRef nodeRef, QName propertyQName, boolean upd
      */
     @Auditable(parameters = {"nodeRef", "expiresAt"})
     public DirectAccessUrl getDirectAccessUrl(NodeRef nodeRef, Date expiresAt);
+
+    /**
+     * Checks if the store supports the retrieving of direct access URLs.
+     *
+     * @return true if direct access URLs retrieving is supported, false otherwise
+     */
+    default boolean isDirectAccessSupported()
```

Comment:
I think this one is part of the other feature :)

Reply:
Thank you, updated :)